DS and Algorithm/Selenium/WebDriverSetup.java

```diff
@@ -0,0 +1,10 @@
+// src/main/java/WebDriverSetup.java
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+public class WebDriverSetup {
+    public static WebDriver initializeChromeDriver() {
+        System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/6#issuecomment-2161300521>#267f6c</a><span> - 06/11/2024, 05:45 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Hardcoding the path to the ChromeDriver executable can lead to issues when running the code on different environments or systems where the path might be different.
</div>
<div id="fix">
<b>Fix:</b> Use a configuration file or environment variable to set the path to the ChromeDriver executable. This makes the code more flexible and portable across different environments.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div>
<pre><code>
 @@ -6,7 +6,7 @@
  public static WebDriver initializeChromeDriver() {
 -        System.setProperty("webdriver.chrome.driver", "lib/chromedriver.exe");
 +        System.setProperty("webdriver.chrome.driver", System.getenv("CHROME_DRIVER_PATH"));
          return new ChromeDriver();
      }
  }
</code></pre>
</div>
</details>
</div>
<br>


</div>